src/types.db

```diff
@@ -161,6 +161,8 @@ power			value:GAUGE:0:U
 pressure			value:GAUGE:0:U
 protocol_counter	value:DERIVE:0:U
 ps_code			value:GAUGE:0:9223372036854775807
+ps_cswitch_vol          value:DERIVE:0:U
```

Comment:
Please use the existing `contextswitch` type – once with *type instance* set to `voluntary` and once set to `involuntary`.